kotlinx.fuzz.jazzer/src/main/kotlin/kotlinx/fuzz/jazzer/JazzerEngine.kt

```diff
@@ -90,49 +97,45 @@ class JazzerEngine(private val config: KFuzzConfig) : KFuzzEngine {
     }
 
     override fun finishExecution() {
-        val directoryPath = Path(Opt.reproducerPath.get()).toAbsolutePath().toString()
-        val dir = File(directoryPath)
-
-        val stacktraceFiles = dir.listFiles { file -> file.name.startsWith("stacktrace-") }
-            ?: throw IllegalStateException("Unable to list files in directory: $directoryPath")
+        val directoryPath = Paths.get(Opt.reproducerPath.get()).toAbsolutePath()
+        val stacktraceFiles = directoryPath.listDirectoryEntries()
+            .filter { it.fileName.toString().startsWith("stacktrace-") }
+            .toList()
 
         val rawStackTraces = mutableListOf<String>()
-        val fileMapping = mutableListOf<Pair<String, String>>()
+        val fileMapping = mutableListOf<Pair<Path, Path>>()
 
         stacktraceFiles.forEach { file ->
-            val crashFile = File(dir, "crash-${file.name.removePrefix("stacktrace-")}")
-            val lines = convertToJavaStyleStackTrace(file.readText())
+            val crashFile = directoryPath.resolve("crash-${file.fileName.toString().removePrefix("stacktrace-")}")
```

Comment:
You can use `Path.name` instead of `fileName.toString`. Literally the same:
```kotlin
@SinceKotlin("1.5")
@WasExperimental(ExperimentalPathApi::class)
public val Path.name: String
    get() = fileName?.toString().orEmpty()
```

Reply:
Fixed